src/plugins/mission/mission_impl.cpp

```diff
@@ -267,7 +267,9 @@ float MissionImpl::hold_time(const MissionItem& item)
 float MissionImpl::acceptance_radius(const MissionItem& item)
 {
     float acceptance_radius_m;
-    if (item.is_fly_through) {
+    if (std::isfinite(item.acceptance_radius_m)) {
+        acceptance_radius_m = item.acceptance_radius_m;
+    } else if (item.is_fly_through) {
         // _acceptance_radius_m is 0, determine the radius using fly_through
```

Comment:
Not related to this PR, but I don't get this comment. @julianoes would you have some insights about it?

Reply:
Me neither :thinking: